package.json

```diff
@@ -50,16 +50,17 @@
       "plugin:shopify/prettier"
     ]
   },
+  "stylelint": {
+    "extends": [
+      "stylelint-config-shopify"
```

Comment:
You should extend from  "stylelint-config-shopify/prettier" if you want to enable running prettier as part of stylelint